sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -343,7 +344,11 @@ export class ManagementClient extends LinkEntity {
    * @returns Promise<ReceivedSBMessage[]>
    */
   async peek(messageCount?: number): Promise<ReceivedMessageInfo[]> {
-    throwErrorIfConnectionClosed(this._context.namespace);
+    throwErrorIfClientOrConnectionClosed(
+      this._context.namespace,
+      this._context.entityPath,
+      this._context.isClosed
```

Comment:
Thought it was myself who defined this helper method a year ago, it is yelling at me now that we might have just passed in the entire context :(

(we dont need to refactor it now, I want to keep the changes minimal to Track 1)